playbooks/roles/ses-osh/tasks/main.yml

```diff
@@ -11,6 +11,15 @@
   with_items:
     - "{{ hostvars[groups['ses_nodes'][0]].ceph_files.results }}"
 
+- name: Get key values for ses-config.yml
+  set_fact:
+    "ceph_{{ item | regex_replace('[\\.\\-]', '_') }}_key":
```

Comment:
Where is this fact used? I am confused why this needs to be a set fact.

Reply:
I tried to leave most of the original PR (36) intact. These variables are used in the ses_config.yml.j2 template when constructing the new ses_config file, and they were originally based on the names of the ses_openstack_keys in the ses-ansible role, which are hard coded to "client.glance", "client.cinder", and "client.cinder_backup". However, those keys don't seem to be accessible from the ses-osh role, so I simply made a list of the names. I'm certainly open to changing / improving this, however.

Reply:
Oh yeah I see now. 

Wouldn't it be simpler to do the lookups directly in the template? It would be far easier to read/parse.